### PR TITLE
Sort change callback definition

### PR DESCRIPTION
**Pull request by asdiallo**

### CODE REVIEW

**Comment by vvakame** [comment]
*ui-grid/ui-grid.d.ts*

to authors(@btesser @joeskeen). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

File: ui-grid/ui-grid.d.ts [hunk 1]
```diff
@@ -2324,6 +2330,15 @@ declare module uiGrid {
...
             (newPage: number, pageSize: number): void;
         }
+
+        export interface sortChangedHandler {
+            /**
+             * The callback for sortChanged
+             * @param {uiGrid.IGridInstance} grid the grid instance
+             * @param {Array<uiGrid.IGridColumn>} sortColumns sorting columns
+             */
+            (grid: uiGrid.IGridInstance, sortColumns: Array<uiGrid.IGridColumn>): void;
```
Comment (joeskeen) [inline ui-grid/ui-grid.d.ts]: I know this PR was opened before #5972 was merged, but I think we should use the generic versions of `IGridInstance` and `IGridColumn` here to be consistent and provide type helps when using this feature.
In order to do this, change: 
`uiGrid.IGridInstance` to `uiGrid.IGridInstanceOf<TEntity>`,
`uiGrid.IGridColumn` to `uiGrid.IGridColumnOf<TEntity>`,
`sortChangedHandler` to `sortChangedHandler<TEntity>`,
then add generics up the chain as necessary.

**Comment by joeskeen** [comment]
Also, it would be good to add a test for this functionality.

**Comment by joeskeen** [comment]
Possible duplicate: #5901

**Comment by MatejQ** [comment]
Can this be closed now that #6352 is merged?

**Comment by asdiallo** [comment]
Yes

-----Message d'origine-----
De : "MatejQ" <notifications@github.com>
Envoyé : ‎23/‎10/‎2015 14:01
À : "borisyankov/DefinitelyTyped" <DefinitelyTyped@noreply.github.com>
Cc : "asdiallo" <abdoulsalamy@yahoo.fr>
Objet : Re: [DefinitelyTyped] Sort change callback definition (#6076)

Can this be closed now that #6352 is merged?

**Pull request closed without merging**